armi/cases/inputModifiers/tests/test_inputModifiers.py

```diff
@@ -200,7 +200,7 @@ def SuiteNaming(index, _case, _mods):
                 f"case-suite-testBPBM/000{case_nbr}/armi-000{case_nbr}-blueprints.yaml",
                 "r",
             )
-            bp_dict = yaml.load(yamlfile)
+            bp_dict = yaml.YAML(typ="unsafe", pure=True).load(yamlfile)
```

Comment:
What is this "unsafe" about?

Reply:
It's been hard to find. I dont know why the deprecation warning printed that out. 

From the [docs](https://yaml.readthedocs.io/en/latest/basicuse.html): 

> typ='safe' accomplishes the same as what safe_load() did before: loading of a document without resolving unknown tags. Provide pure=True to enforce using the pure Python implementation, otherwise the faster C libraries will be used when possible/available but these behave slightly different (and sometimes more like a YAML 1.1 loader).

Seems like I could remove both the `typ` and `pure` args and it would all run fine.

Reply:
I can't load the docstring, but the answer here explains the different options. Seems like we want just the default behavior though. https://stackoverflow.com/questions/51316491/ruamel-yaml-clarification-on-typ-and-pure-true